service/routes.py

```diff
@@ -73,7 +79,28 @@ def create_product():
     return (jsonify(product.serialize()), status.HTTP_201_CREATED)
 
 
-# todo - {"Location": location_url} << we have to addd this later
+# todo - {"Location": location_url} << we have to add this later
+
+
+############################################################
+# Update counters
+############################################################
+@app.route("/products/<name>", methods=["PUT"])
+def update_product(name):
+    """Update a counter"""
+    app.logger.info("Request to Update counter: '%s'...", name)
+
+    # check if the product exists
+    counter = Product.find(name)
+    if counter is None:
+        error(status.HTTP_404_NOT_FOUND, f"Counter '{name}' does not exist")
+
+    count = counter.increment()
+
```

Comment:
```suggestion
counter.update()
```
There is a counter.update() line that is missing